backend/src/Altinn.Authorizationadmin/Altinn.Authorizationadmin/Services/Implementation/PolicyAdministrationPoint.cs

```diff
@@ -309,12 +311,23 @@ private async Task<List<Rule>> ProcessPolicyFile(string policyPath, string org,
                 return null;
             }
 
+            if (!string.IsNullOrWhiteSpace(org) || !string.IsNullOrWhiteSpace(app))
+            {
+                altinnAppId = $"{org}/{app}";
+            }
+
+            resource = await _resourceRegistryClient.GetResource(resourceregistryId);
```

Comment:
Resource lookup should only be performed if resourceRegistryId has a value

Reply:
but also unsure whether we need to actually lookup the resource in the registry for delete operation. 
Should perhaps be enough that the delegation policy exist and we are able to get a lease lock for the blob?